content/updates/2020-10/6-Job-and-process-tracker-improvements.md

```diff
@@ -1,22 +1,40 @@
 ---
-pim_announcement_alt_img: ""
+pim_announcement_alt_img: "TODO"
 pim_announcement_audience:
 - EE
 - CE
 ---
 
 # Jobs and process tracker improvements
 
-TODO
+Follow the estimated time left for your ongoing jobs, quickly jump to the process tracker once you launched a quick export, and enjoy some enhancements directly inside your process tracker grid.
```

Comment:
```suggestion
We made several enhancements to our process tracker: we now display a progress bar with the estimated remaing time before your job is completed, a shortcut to the process tracker when you launch a quick export and we added a new filter in the process tracker grid.
```

Reply:
Sentence construction is weird at the end. You have 3 things but only 2 verbs. And there is a typo at "remaining". ;)

Reply:
I've rearranged the text and fixed typos :)
```suggestion
Check our latest process tracker enhancements: a progress bar and the estimated remaining time for your ongoing jobs, a shortcut for a rapid access to quick exports job details, and a new sorting option in the process tracker grid.
```